contribution-guidelines.md

```diff
@@ -0,0 +1,7 @@
+# Contribution guidelines
+
+Hier entsteht das Free and Open Source-Software-Konzept der grün-roten Regierungskoalition im Münchner Rathaus.
+Wir freuen uns über Eure Teilnahme und Euer Feedback!
```

Comment:
Ah, das müssen wir ändern, Lars Mentrup hat sich ja beim letzten Treffen beschwert, dass das kein offizielles Repo ist und wir das kennzeichnen müssen. Also bitte noch was in der Art schreiben, dass das unsere Interpretation/Ausgestaltung des Koalitionsvertrags ist. :)

Reply:
@missgreenwood Du bist verantwortlich für den Inhalt und sehe Dich in der Rollen als [Maintainer](https://de.wikipedia.org/wiki/Maintainer) (qua Amt).
Ist das so ok?

Reply:
Als Link hab ich mal den von [muenchen.de](https://www.muenchen.de/rathaus/Stadtpolitik/Der-Muenchner-Stadtrat/Stadtratsmitglieder/2020-2026/Greif.html) genommen, das offiziellste was ich so gefunden habe.
Kann man auch was anderes nutzen.

Reply:
Passt super, vielen lieben Dank.